src/write_hdf5_xml/write_hdf5_xml.c

```diff
@@ -2659,6 +2696,10 @@ int main(int argc, char **argv)
       sprintf(data_source, "ALOS PALSAR");
       sprintf(copyright, "JAXA, METI (%d)", year);
     }
+    if (strncmp_case(meta->general->sensor, "ERS", 3) == 0) {
```

Comment:
This could be "else if"
and we would want a catch-all else case:
else {
   asfPrintError("Unable to set copyright for sensor: %s!", meta->general->sensor);
}